### DISCUSSION TITLE
Audio "bad request"

### DISCUSSION THREAD

**Discussion by clothstir** [Lednerb/bilberry-hugo-theme#405] [Q&A] [answered]
I see on the demo site (https://lednerb.github.io/bilberry-hugo-theme/audio/kilez-more-new-world-order/) the same problem I see on my site with the demo material, it says "bad request." I assume it is supposed to include a player or embed of some kind?

Thanks in advance!

**Comment by igor-baiborodine** [comment, marked as answer]
There was an issue with the Spotify track ID. Now it's fixed.
